dags/pytorch_xla/configs/pytorchxla_torchbench_config.py

```diff
@@ -85,16 +85,16 @@ class R2_4(enum.Enum):
     TORCH_XLA_REPO_BRANCH = "-b v2.4.0-rc8"
 
   class R2_5(enum.Enum):
-    TORCH_XLA_TPU_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.5.0rc10-cp310-cp310-linux_x86_64.whl"
-    TORCH_XLA_CUDA_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0rc10-cp310-cp310-linux_x86_64.whl"
+    TORCH_XLA_TPU_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.5.0%2Blibtpu-cp310-cp310-manylinux_2_28_x86_64.whl"
```

Comment:
Also update this ` libtpu`. Then we can launch the final test :>

Reply:
Thanks, nice catch, updated